src/ray/core_worker/reference_count.cc

```diff
@@ -33,6 +104,8 @@ void ReferenceCounter::AddLocalReference(const ObjectID &object_id) {
     it = object_id_refs_.emplace(object_id, Reference()).first;
   }
   it->second.local_ref_count++;
+  RAY_LOG(DEBUG) << "Add local reference " << object_id;
+  PRINT_REF_COUNT(it);
```

Comment:
This seems like it's going to be extremely noisy if I ever turn on DEBUG mode. Not sure we want these here by default

Reply:
Hmm I think it's fine since we won't call them for Python references, but we can remove them later if they're annoying.